app/controllers/shortlists_controller.rb

```diff
@@ -13,9 +13,19 @@ def show
 private
 
   def filter_params
-    p = params.permit(filter: { placement_types: [] })[:filter]
+    p = params.permit(filter: {
+      search_radius: "",
+      latitude: "",
+      longitude: "",
+      placement_types: [],
+    })[:filter]
     if p.nil? && @placement_need
-      { placement_types: [@placement_need.criteria] }
+      {
+        placement_types: [@placement_need.criteria],
+        search_radius: @placement_need.location_radius_miles,
```

Comment:
I actually just realised that this is going to cause problems with my work in progress 🤔  I'm moving this code completely around to persist on the `Shortlist`. Have you added these filters here, because you anticipate this to be in the filter on the left? 
If you don't mind, I would put it there only when the filter is actually implemented, because it will have to be persisted on the `Shortlist` after I've finished.